package.json

```diff
@@ -25,29 +25,29 @@
   "dependencies": {
     "chalk": "~1.1.1",
     "insight": "~0.7.0",
-    "lodash": "~3.10.1",
+    "lodash": "~4.3.0",
     "slash": "~1.0.0",
     "underscore.string": "~3.2.2",
-    "yeoman-generator": "~0.20.3",
-    "yosay": "~1.0.5"
+    "yeoman-generator": "~0.22.5",
+    "yosay": "~1.1.0"
   },
   "devDependencies": {
-    "bluebird": "~2.10.2",
-    "bower": "~1.5.3",
-    "chai": "~3.3.0",
-    "chai-as-promised": "~5.1.0",
-    "commander": "~2.8.1",
-    "cross-spawn": "~2.0.0",
-    "ejs": "~2.3.4",
+    "bluebird": "~3.2.2",
+    "bower": "~1.7.7",
+    "chai": "~3.5.0",
+    "chai-as-promised": "~5.2.0",
+    "commander": "~2.9.0",
+    "cross-spawn": "~2.1.5",
+    "ejs": "~2.4.1",
     "eslint": "~1.6.0",
-    "istanbul": "~0.3.22",
-    "js-beautify": "~1.5.10",
+    "istanbul": "~0.4.2",
+    "js-beautify": "~1.6.2",
     "mkdirp": "~0.5.1",
-    "mocha": "~2.3.3",
+    "mocha": "~2.4.5",
     "mocha-lcov-reporter": "~1.0.0",
-    "mz": "~2.0.0",
-    "recursive-readdir": "~1.2.1",
-    "sinon": "~1.17.1",
+    "mz": "~2.3.1",
+    "recursive-readdir": "~1.3.0",
+    "sinon": "~1.17.3",
```

Comment:
Just going off of the depedency monitor. I'm unopininated about it. https://david-dm.org/eleven-labs/generator-gulp-angular#info=devDependencies&view=table